science/libsbml/Portfile

```diff
@@ -18,32 +21,117 @@ long_description    LibSBML is a free, open-source programming library to \
                     rather a library you can embed in your own applications.
 
 homepage            http://sbml.org/Software/libSBML
-master_sites        sourceforge:project/sbml/libsbml/${version}/stable
+master_sites        sourceforge:project/sbml/libsbml/${version}/experimental/source
+patchfiles          patch-python-mp.diff
+
+distname            libSBML-${version}-Source
+
+checksums           rmd160  e331901fdc87a733cdc113e83b1d3eaa27a76538 \
+                    sha256  4a2114ce1b48ff19f9be088ce75be57d54c262541a464775929e1cc66e9c943b
+
+depends_lib-append  port:bzip2 \
+                    port:zlib \
+                    port:pkgconfig
+
+configure.args-append -DWITH_EXAMPLES:BOOL=ON
+
+variant cppnamespace description {Use a C++ namespace for libSBML} {
+    configure.args-append   -DWITH_CPP_NAMESPACE:BOOL=ON
+}
+
+variant arrays description {Enable libSBML support for the SBML Level 3 Arrays and Sets ('arrays')} {
+    configure.args-append   -DENABLE_ARRAYS:BOOL=ON
+}
+
+variant comp description {Enable libSBML support for the SBML Level 3 Hierarchical Model ('comp')} {
+    configure.args-append   -DENABLE_COMP:BOOL=ON
+}
```

Comment:
It's nitpicky, but please keep the variants sorted by name.

Reply:
Thank you for your comment.
I've changed the Portfile to use cmake 1.1 and sorted the variants by name.